### PR TITLE
RWA-107: Setting up work-allocation s2s secrets

### PR DESCRIPTION
**Pull request by alexbaeza**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RWA-107

### Change description ###

```
❯ ./check-all-vaults-for-s2s-secret wa-workflow-api                   
INFO: Found secret wa-workflow-api in environment demo
INFO: Found secret wa-workflow-api in environment aat
INFO: Found secret wa-workflow-api in environment prod
INFO: Found secret wa-workflow-api in environment perftest
INFO: Found secret wa-workflow-api in environment ithc



❯ ./check-all-vaults-for-s2s-secret wa-task-management-api                                                                 
INFO: Found secret wa-task-management-api in environment demo
INFO: Found secret wa-task-management-api in environment aat
INFO: Found secret wa-task-management-api in environment prod
INFO: Found secret wa-task-management-api in environment perftest
INFO: Found secret wa-task-management-api in environment ithc



❯ ./check-all-vaults-for-s2s-secret wa-task-configuration-api                                                              
INFO: Found secret wa-task-configuration-api in environment demo
INFO: Found secret wa-task-configuration-api in environment aat
INFO: Found secret wa-task-configuration-api in environment prod
INFO: Found secret wa-task-configuration-api in environment perftest
INFO: Found secret wa-task-configuration-api in environment ithc

```
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/resources/bootstrap.yaml [hunk 1]
```diff
@@ -81,5 +81,7 @@ spring:
         s2s.microservicekey-aac-manage-case-assignment: microserviceKeys.aac_manage_case_assignment
         s2s.microservicekey-unspec-service: microserviceKeys.unspec_service
         s2s.microservicekey-rd-location-ref-data-load: microserviceKeys.rd_location_ref_data_load
-        s2s.microservicekey-wa: microserviceKeys.wa
-        s2s.microservicekey-rd-location-ref-api: microserviceKeys.rd_location_ref_api
\ No newline at end of file
+        s2s.microservicekey-wa-workflow-api: microserviceKeys.wa_workflow_api
+        s2s.microservicekey-wa-task-management-api: microserviceKeys.wa_task-management_api
+        s2s.microservicekey-wa-task-configuration-api: microserviceKeys.wa_task-configuration_api
```
Comment (raghera) [inline src/main/resources/bootstrap.yaml:86]: should all be `underscores`

File: src/main/resources/bootstrap.yaml [hunk 2]
```diff
@@ -81,5 +81,7 @@ spring:
         s2s.microservicekey-aac-manage-case-assignment: microserviceKeys.aac_manage_case_assignment
         s2s.microservicekey-unspec-service: microserviceKeys.unspec_service
         s2s.microservicekey-rd-location-ref-data-load: microserviceKeys.rd_location_ref_data_load
-        s2s.microservicekey-wa: microserviceKeys.wa
-        s2s.microservicekey-rd-location-ref-api: microserviceKeys.rd_location_ref_api
\ No newline at end of file
+        s2s.microservicekey-wa-workflow-api: microserviceKeys.wa_workflow_api
+        s2s.microservicekey-wa-task-management-api: microserviceKeys.wa_task-management_api
```
Comment (raghera) [inline src/main/resources/bootstrap.yaml:85]: underscores

File: src/main/resources/bootstrap.yaml (hunk 1 above)
Comment (alexbaeza) [reply src/main/resources/bootstrap.yaml]: Thanks :)

**Review by adusumillipraveen — COMMENTED** [review]
can you pls stick to this template : https://github.com/hmcts/service-auth-provider-app/blob/master/.github/PULL_REQUEST_TEMPLATE.md#if-youre-adding-a-new-secret-then-do-the-following 

(run the script and update description)

**Comment by alexbaeza** [comment]
> can you pls stick to this template : https://github.com/hmcts/service-auth-provider-app/blob/master/.github/PULL_REQUEST_TEMPLATE.md#if-youre-adding-a-new-secret-then-do-the-following
> 
> (run the script and update description)

done 👍

**Review by adusumillipraveen — APPROVED** [review]

**Review by raghera — APPROVED** [review]

**Pull request merged**